src/settings/toml/manifest.rs

```diff
@@ -43,6 +43,7 @@ pub struct Manifest {
     pub kv_namespaces: Option<Vec<ConfigKvNamespace>>,
     pub env: Option<HashMap<String, Environment>>,
     pub vars: Option<HashMap<String, String>>,
+    pub blobs: Option<HashMap<String, String>>,
```

Comment:
you should be able to use a `<HashMap<String, PathBuf>>` explicitly

Reply:
I'd have used PathBuf, but webpack_config above uses String so I used that, but I can change it if you'd like.

Reply:
yes please 😄